RealTimeFaceMaskDetector/LogViewer/LogSearcher.h

```diff
@@ -18,6 +18,12 @@ class LogSearcher
 
 		const std::string EXE_DIRECTORY = ExeDirectoryPath();
 		const std::string LOG_DIRECTORY = EXE_DIRECTORY + "\\RTFMD_Logs";
+
+		if (!fs::exists(LOG_DIRECTORY))
```

Comment:
Let's make it with if(fs::exits(LOG_DIRECTORY))

Reply:
Thank you for the suggestion. I've made it like that.